api/core/tools/provider/builtin/agentql/tools/query_data.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from typing import Any, Union
+
+import httpx
+
+from core.tools.entities.tool_entities import ToolInvokeMessage
+from core.tools.tool.builtin_tool import BuiltinTool
+
+logger = logging.getLogger(__name__)
+
+
+class QueryDataTool(BuiltinTool):
+    def _invoke(
+        self,
+        user_id: str,
+        tool_parameters: dict[str, Any],
+    ) -> Union[ToolInvokeMessage, list[ToolInvokeMessage]]:
+        """
+        invoke tools
+        """
+        url = tool_parameters["url"]
+        query = tool_parameters["query"]
+        timeout = tool_parameters["timeout"]
+
+        params = {}
+        params["mode"] = tool_parameters.get("mode", "fast")
```

Comment:
Do I still need to provide default values here or can I just rely on default values in yaml config?